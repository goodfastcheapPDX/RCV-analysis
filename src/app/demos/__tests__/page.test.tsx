import { render } from "@testing-library/react";
import { describe, expect, it } from "vitest";
import Demo from "../page";

describe("Demo Page", () => {
  it("should match snapshot", () => {
    const { container } = render(<Demo />);
    expect(container.firstChild).toMatchSnapshot();
  });
});
