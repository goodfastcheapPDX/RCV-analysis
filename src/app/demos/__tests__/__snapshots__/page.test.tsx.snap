// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Demo Page > should match snapshot 1`] = `
<div
  class="flex gap-4 items-center flex-col sm:flex-row h-screen w-screen justify-center"
>
  <a
    class="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
    href="/"
  >
    Home
  </a>
  <a
    class="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
    href="/demos/first-choice"
  >
    First Choice Votes Chart
  </a>
  <a
    class="rounded-full border border-solid border-transparent transition-colors flex items-center justify-center bg-foreground text-background gap-2 hover:bg-[#383838] dark:hover:bg-[#ccc] font-medium text-sm sm:text-base h-10 sm:h-12 px-4 sm:px-5 sm:w-auto"
    href="/demos/stv-rounds"
  >
    Animated Round-by-round Election Data
  </a>
</div>
`;
